src/dlload.c

```diff
@@ -57,6 +57,30 @@ static int endswith_extension(const char *path) JL_NOTSAFEPOINT
     return 0;
 }
 
+#ifdef _OS_WINDOWS_
+#ifdef _MSC_VER
+#if (_MSC_VER >= 1930) || (_MSC_VER < 1800)
+#error This version of MSVC has not been tested.
+#define CRTDLL_BASENAME "msvcrt"
+#elif _MSC_VER >= 1920 // VC++ 2019
+#define CRTDLL_BASENAME "msvcr142"
+#elif _MSC_VER >= 1910 // VC++ 2017
+#define CRTDLL_BASENAME "msvcr141"
+#elif _MSC_VER >= 1900 // VC++ 2015
+#define CRTDLL_BASENAME "msvcr140"
```

Comment:
I don't think these will work well in practice. So I'm going to remove them.

Microsoft is thinking of migrating to Universal CRT.
https://docs.microsoft.com/en-us/cpp/windows/universal-crt-deployment?view=msvc-160

Also, it seems that "msvcr140.dll" has been replaced by "vcruntime140.dll".

I don't think it's a good idea to keep using "msvcrt.dll" forever, but the situation is chaotic.